### PR TITLE
Cisco firepower e streamer cef

### PR DESCRIPTION
**Pull request by skhademcis**
Fixes #

## Proposed Changes

  -  Initial 4.0 Deployment of eNcore Connector for Azure Sentinel

### CODE REVIEW

**Comment by erant10** [comment]
@skhademcis I just notice one other thing - Looks like links you added are hard coded into the instruction text field. It would be better if these links were formatted using markdown like so: \[this is a link\](https://example.com) 

For example in step 2 - 

Install and configure the Firepower eNcore eStreamer client, full guide is \[located here\]( https://www.cisco.com/c/en/us/td/docs/security/firepower/670/api/eStreamer_enCore/eStreamereNcoreSentinelOperationsGuide_409.html)

this will make `located here` be displayed as a link and the url will not be visible as plain text

File: DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        {
+            "title": "2. Install the Firepower eNcore client",
+            "description": "Install and configure the Firepower eNcore eStreamer client, full guide is located here https://www.cisco.com/c/en/us/td/docs/security/firepower/670/api/eStreamer_enCore/eStreamereNcoreSentinelOperationsGuide_409.html",
+            "innerSteps": [
+                {
+                    "title": "2.1 Download the Firepower Connector from github",
+                    "description": "Download the latest version of the Firepower eNcore connector for Microsoft Sentinel here https://github.com/CiscoSecurity/fp-05-microsoft-sentinel-connector.  If you plan on using python3 use the python3 branch here https://github.com/CiscoSecurity/fp-05-microsoft-sentinel-connector/tree/python3"
+                },
+                {
+                    "title": "2.1 Create a pkcs12 file using the Azure/VM Ip Address",
```
Comment (erant10) [inline DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json:97]: step 2.1 appears twice.

File: DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                                "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
+                            },
+                            "type": "CopyableLabel"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "title": "2. Forward Common Event Format (CEF) logs to Syslog agent",
```
Comment (erant10) [inline DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json:128]: this should be step 3 right?

File: DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json (hunk 1 above)
Comment (skhademcis) [reply DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json]: should be updated in the latest

File: Logos/cisco-logo-72px.svg [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        <g transform="translate(0.000000, 15.000000)" fill="#049FD9">
+            <polygon id="Fill-1" points="21 28 24 28 24 42 21 42"></polygon>
+            <path d="M50,32.0833333 C49.8837209,31.9666667 48.8372093,31.3833333 47.2093023,31.3833333 C45.1162791,31.3833333 43.6046512,32.9 43.6046512,35 C43.6046512,36.9833333 45,38.6166667 47.2093023,38.6166667 C48.7209302,38.6166667 49.8837209,38.0333333 50,37.9166667 L50,41.5333333 C49.5348837,41.65 48.4883721,42 46.9767442,42 C43.255814,42 40,39.3166667 40,35 C40,31.0333333 42.9069767,28 46.9767442,28 C48.4883721,28 49.6511628,28.35 50,28.4666667 L50,32.0833333" id="Fill-2"></path>
+            <path d="M16,32.0833333 C15.8837209,31.9666667 14.8372093,31.3833333 13.2093023,31.3833333 C11.1162791,31.3833333 9.60465116,32.9 9.60465116,35 C9.60465116,36.9833333 11,38.6166667 13.2093023,38.6166667 C14.7209302,38.6166667 15.8837209,38.0333333 16,37.9166667 L16,41.5333333 C15.5348837,41.65 14.4883721,42 12.9767442,42 C9.25581395,42 6,39.3166667 6,35 C6,31.0333333 8.90697674,28 12.9767442,28 C14.4883721,28 15.6511628,28.35 16,28.4666667 L16,32.0833333" id="Fill-3"></path>
```
Comment (erant10) [inline Logos/cisco-logo-72px.svg:9]: Also please replace all of the `Fill-x` ids with [unique GUIDs ](https://www.guidgenerator.com), or remove them entirely if they are not needed

**Review by erant10 — DISMISSED** [review]

**Comment by erant10** [comment]
@skhademcis Please Sign the CLA to pass all checks

**Review by erant10 — COMMENTED** [review]
please sign the cla. we can't merge the PR without it

**Review by nazang — CHANGES_REQUESTED** [review]
@skhademicis,
 Could you sign the cla? We can only merge the request after you signed the license.
 Please update the branch with latest master as well: This branch is out-of-date with the base branch

File: Logos/cisco-logo-72px.svg [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <desc>Created with Sketch.</desc>
```
Comment (v-jayakal) [inline Logos/cisco-logo-72px.svg:4]: Can you please remove the attribute <desc> completely. Its invalid attribute.

File: Logos/cisco-logo-72px.svg [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (v-jayakal) [inline Logos/cisco-logo-72px.svg:2]: Can you please remove the invalid version attribute?

File: Logos/cisco-logo-72px.svg [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <desc>Created with Sketch.</desc>
+    <g id="cisco_sentinel_connector_logo_svg" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="cisco_sentinel_connector_logo" transform="translate(0.000000, 15.000000)" fill="#049FD9">
```
Comment (v-jayakal) [inline Logos/cisco-logo-72px.svg:6]: Please provide guids so that these are uniquely identifiable across all Azure logo assets

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: Logos/cisco-logo-72px.svg (hunk 6 above)
Comment (skhademcis) [reply Logos/cisco-logo-72px.svg]: Completed

File: Logos/cisco-logo-72px.svg (hunk 4 above)
Comment (skhademcis) [reply Logos/cisco-logo-72px.svg:4]: Completed

Comment (v-jayakal) [reply Logos/cisco-logo-72px.svg:4]: Can you please remove  "<desc>Created with Sketch.</desc>"?

Comment (v-jayakal) [reply Logos/cisco-logo-72px.svg:4]: Remove desc element ( desc-Created with Sketch-desc).

File: Logos/cisco-logo-72px.svg [hunk 7]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (nazang) [inline Logos/cisco-logo-72px.svg:2]: Remove xmlns:xlink="http://www.w3.org/1999/xlink" from .svg file. 

Guidance is available at:

https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/ReadMe.md

Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name

**Review by nazang — CHANGES_REQUESTED** [review]

File: Logos/cisco-logo-72px.svg (hunk 7 above)
Comment (skhademcis) [reply Logos/cisco-logo-72px.svg]: complete

**Review by v-jayakal — APPROVED** [review]

**Review by nazang — APPROVED** [review]

**Pull request merged**